applications/vila_live/README.md

```diff
@@ -0,0 +1,37 @@
+# Holoscan VILA Live
```

Comment:
Was this inspired by any other projects? (Jetson?) If so, could you please add a references section at the bottom of the README with link(s)?

Reply:
Added links